### ISSUE TITLE
[BUG] - I cannot get HomeAssistant to see the image for meal of the day through Generic Camera

### ISSUE THREAD

**Issue by GavinKABarnes** [mealie-recipes/mealie#4232]
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

I'm trying to integrate Mealie into HomeAssistant, I have set up the sensors correctly in the YAML file and they are working correctly, however, when I use the link in the documentation (with my IP and port number), I get the message "Unable to load valid image from still image URL: The URL was not found on the server."

Mealie is hosted through my Unraid server, and all other functionality seems to work.

### Steps to Reproduce

I'm not sure how to have this reproduced as I can't seem to find anyone who's had this issue.

### Please provide relevant logs

I'm not sure what logs I could even put in here, as the docker runs fine, I just cannot access the meal of the day image

### Mealie Version

ghcr.io/mealie-recipes/mealie:nightly

### Deployment

Unraid

**Comment by GavinKABarnes** [comment]
Upon further inspection, it seems that the recipe ID/Meal ID sensor is reporting "unknown", which is likely what is causing the issue

I'm not entirely sure why, however, here's my YAML config for that sensor

- platform: rest
  resource: "http://MyIP:MyPort/api/households/mealplans/today"
  method: GET
  name: Mealie todays meal ID
  headers:
    Authorization: Bearer My API Token
  value_template: "{{ value_json[0].recipe.id }}"
  force_update: true
  scan_interval: 30

**Comment by mopedy** [comment]
Hi, I had the same, changing households to groups resolved this for me.

- platform: rest
  resource: "http://192.168.1.116:9925/api/groups/mealplans/today"
  method: GET
  name: Mealie todays meal ID

**Comment by michael-genson** [comment]
> changing households to groups resolved this for me

If you're on v1.12 or earlier, this is correct. If you're on nightly (or in the future) households is correct

**Comment by skank01** [comment]
Got same issue, 
Checked version, i'm on nightly, so i changed sensor to use households, but still the same.
No image, also not with groups.

Anyone has correct settings for the generic one?

**Comment by skank01** [comment]
Nevermind, fixed ! url was wrong

**Comment by michaelcox12** [comment]
Was it just the IP or the port number? Seems like for the life of me I can't get the sensors to read anything except "unknown"

I followed this page to a tee, rechecked IP/Port, assigned meals to every day of the week, can't get the sensor to show a value except Unknown in the Developer tools in HA

https://docs.mealie.io/documentation/community-guide/home-assistant/

Is there a way to troubleshoot this? The rest code in my config files shows no errors and I can see the sensors in Dev tools, but can't get a value. Tried a new API token, tried groups instead of household, restarted HA completely between every change, just not sure what could be the issue.

![image](https://github.com/user-attachments/assets/4f06680f-79fe-43be-8695-08f44f08b8cf)

On nightly, latest version I believe

rest:
  - resource: "http://192.168.4.111:9925/api/households/mealplans/today"
    method: GET
    headers:
      Authorization: Bearer EXACT API CODE FROM MEALIE HERE
    scan_interval: 3600
    sensor:
      - name: Mealie todays meal
        value_template: "{{ value_json[0]['recipe']['name'] }}"
        force_update: true
        unique_id: mealie_todays_meal
      - name: Mealie todays meal ID
        value_template: "{{ value_json[0]['recipe']['id'] }}"
        force_update: true
        unique_id: mealie_todays_meal_id
